### ISSUE TITLE
On the zeromenu, site owner is excluded from peer counter

### ISSUE THREAD

**Issue by Plasmmer** [HelloZeroNet/ZeroNet#913]

**Comment by mishfit** [comment]
Do you mean the menu inside your site:
<img width="538" alt="zeronet-peers" src="https://cloud.githubusercontent.com/assets/817064/25292167/413aa024-2692-11e7-8cf8-6b163a5fb859.png">

Or the menu inside [ZeroHello](https://github.com/HelloZeroNet/ZeroHello/)?
<img width="354" alt="zerohello-peers" src="https://cloud.githubusercontent.com/assets/817064/25292189/57990270-2692-11e7-928b-4326d86d6a5d.png">

That will help determine whether this is a ZeroNet Sidebar plugin issue or a ZeroHello site issue

**Comment by Plasmmer** [comment]
ZeroNet Sidebar issue.
In my new site I see one more number, in sidebar is less. I think the sidebar isn't counting the user as a peer.

**Comment by mishfit** [comment]
That seems like a good thing to me...you aren't a peer, you are the current instance.

@shortcutme can weigh in, but if you are correctly listed on the globe and the number of real peers is correct, then it probably shouldn't include you.

I also see what looks like an explicit exclusion of "self" (loopback or "127.0.0.1") as a peer in [SiteManager.py](https://github.com/HelloZeroNet/ZeroNet/blob/master/src/Site/SiteManager.py#L158)
```python
peer_blacklist = [("127.0.0.1", config.fileserver_port)]  # Dont add this peers
```

**Comment by HelloZeroNet** [comment]
Thanks for reporting, fixed in d346a532ffc2b9dcabca423ea84cd0c619e206f0
